examples/angular/src/app/app.component.ts

```diff
@@ -1,14 +1,12 @@
 import { Component } from '@angular/core';
 import { RouterOutlet } from '@angular/router';
-
-import { DotcmsLayoutComponent } from './lib/layout/dotcms-layout/dotcms-layout.component';
+import { DotcmsLayoutComponent } from '@dotcms/angular';
 
 @Component({
   selector: 'app-root',
   standalone: true,
   imports: [RouterOutlet, DotcmsLayoutComponent],
   templateUrl: './app.component.html',
-  styleUrl: './app.component.css'
+  styleUrl: './app.component.css',
```

Comment:
Is it necessary OnPush here?

Reply:
I think no, is only an example app to integrate `dotcms/angular` components